### PR TITLE
Feature/courts

### PR DESCRIPTION
**Pull request by bbrown11924**
Added functionality to add parks where games can be hosted along with several tests.  This feature includes the model, form and views necessary for this functionality

### CODE REVIEW

File: pickup/models.py [hunk 1]
```diff
@@ -2,8 +2,14 @@
...
 
 from django.db import models
+from localflavor.us.models import USStateField, USZipCodeField
+from localflavor.us.us_states import STATE_CHOICES
 from django.contrib.auth.models import User
 
+
+class Player(User):
+    pass
+
```
Comment (jgalita1) [inline pickup/models.py:12]: This should be removed.

Comment (bbrown11924) [reply pickup/models.py:12]: resolved

File: pickup/views.py [hunk 2]
```diff
@@ -1,22 +1,25 @@
...
-from django.http import HttpResponse, HttpResponseRedirect
 from django.core.validators import validate_email
 from django.db.utils import IntegrityError
 from django.contrib.auth.views import LoginView, LogoutView
 from django.contrib.auth.decorators import login_required
 from django.contrib.auth import login
+from django.http import HttpResponse, HttpResponseRedirect
 
+# Import models and forms
+from .forms import ParkForm, RegistrationForm
```
Comment (jgalita1) [inline pickup/views.py:11]: There is some duplication here.

File: pickup/views.py [hunk 3]
```diff
@@ -182,3 +196,29 @@ def edit_profile(request):
...
+            except IntegrityError:
+
+                return render(request, 'pickup/add_park.html')
+
+            return HttpResponse("Park has been added!")
+
+    else:
+        form = ParkForm()
+
+    return render(request, 'add_park.html', {'form': form})
```
Comment (jgalita1) [inline pickup/views.py:224]: You should use reverse() here.

File: pickup/views.py [hunk 4]
```diff
@@ -182,3 +193,29 @@ def edit_profile(request):
...
+            except IntegrityError:
+
+                return render(request, reverse('Add Park'))
+
+            return HttpResponse("Park has been added!")
+
+    else:
+        form = ParkForm()
+
+    return render(request, reverse('Add Park'), {'form': form})
```
Comment (jgalita1) [inline pickup/views.py]: Sorry, these shouldn't be reverse since these are template paths! Sorry!

File: pickup/views.py (hunk 3 above)
Comment (jgalita1) [reply pickup/views.py:221]: Nevermind, sorry!

**Review by jgalita1 — APPROVED** [review]
Discussed on 11/1. Fixed a few issues and successfully merged with latest development.

Looks good, approved.

**Pull request merged**